src/uwtools/config/formats/nml.py

```diff
@@ -34,7 +34,8 @@ def _load(self, config_file: OptionalPath) -> dict:
         :param config_file: Path to config file to load.
         """
         with readable(config_file) as f:
-            return f90nml.read(f)
+            config: dict = f90nml.read(f)
+            return config
```

Comment:
This type of change is a little unfortunate. Bleh.

Reply:
Yeah. The problem is that the type of `f90nml.read()` is `Any` because `f90nml` has no type stubs. An unfortunate workaround for now.